.github/scripts/publish_package.sh

```diff
@@ -17,7 +17,7 @@
 set -e
 set -u
 
-echo "//wombat-dressing-room.appspot.com/:_authToken=${NPM_AUTH_TOKEN}" >> ~/.npmrc
+echo "//wombat-dressing-room.appspot.com/:_authToken=${NPM_AUTH_TOKEN}" >> .npmrc
```

Comment:
Can assume that this runs in root? should we explicitly say `/. npmrc`?

Reply:
The file should be added to the root of the repo/project. All the commands are already executed from the root of the repo. So file paths get resolved relative to that directory.